doc/man/motr.7.txt

```diff
@@ -10,19 +10,23 @@ motr - Motr object stor
 
 DESCRIPTION
 -----------
-Motr object stor ['...to be filled']
+This manpage briefly describes the major binaries and utilities available in
+Motr object stor.
 
 Motr utilities
 ~~~~~~~~~~~~~~
 
 *m0d*::
-  Motr file system service daemon.
+  Motr file system service daemon. This command initializes and starts
+  requested Motr file system service as a daemon process in user space.
```

Comment:
```
Motr service daemon. This command initializes and starts requested
Motr service as a daemon process in user space.
```
Remove the "file system". It is not a file system any longer, or it is misleading.

Reply:
Done.